Jenkinsfile

```diff
@@ -10,39 +10,44 @@
 
 node('docker') {
     checkout scm
-    def buildImage = docker.build("my-image:${env.BUILD_ID}", "-f docker/Dockerfile-Jenkins-Build-Env .")
-    buildImage.inside(" -v /var/run/docker.sock:/var/run/docker.sock " ){
-        stage('Prepare') {
-        sh '''
-            git submodule update --init
-            mkdir -p artifacts
-            rm -f artifacts/*
-            '''
-        }
-        stage('Build') {
-			sh '''
-			 ./docker/build.sh amd64
```

Comment:
should we remove the build.sh file, if it is not usable any more?

Reply:
yes, this is why I marked it draft. Still need doc adjustment and removal of some files

Reply:
Clean up done